eng/packages/http-client-csharp/generator/Azure.Generator/test/TestHelpers/MockHelpers.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp;
+using Microsoft.Generator.CSharp.Input;
+using Microsoft.Generator.CSharp.Primitives;
+using Microsoft.Generator.CSharp.Providers;
+using Microsoft.Generator.CSharp.SourceInput;
+using Moq;
+using Moq.Protected;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace Azure.Generator.Tests.TestHelpers
+{
+    internal class MockHelpers
+    {
+        private static readonly string _configFilePath = Path.Combine(AppContext.BaseDirectory, TestHelpersFolder);
+        public const string TestHelpersFolder = "TestHelpers";
```

Comment:
this is not used outside, does it need to be public?

Reply:
this is copied from MGC repo here.
Let me change it to private.